src/app/inventory/d1-stores.ts

```diff
@@ -26,33 +26,39 @@ import { resetItemIndexGenerator } from './store/item-index';
 export function loadStores(): ThunkResult<D1Store[] | undefined> {
   return async (dispatch, getState) => {
     const promise = (async () => {
+      // TODO: check account here
       try {
         let account = currentAccountSelector(getState());
         if (!account) {
           await dispatch(getPlatforms());
           account = currentAccountSelector(getState());
-          if (!account) {
+          if (!account || account.destinyVersion !== 1) {
             return;
           }
         }
+
         resetItemIndexGenerator();
 
         const [defs, , rawStores] = await Promise.all([
           dispatch(getDefinitions()) as any as Promise<D1ManifestDefinitions>,
           dispatch(loadNewItems(account)),
           getStores(account),
         ]);
+
         const lastPlayedDate = findLastPlayedDate(rawStores);
-        const buckets = bucketsSelector(getState())!;
+        const buckets = d1BucketsSelector(getState())!;
```

Comment:
This incidentally fixes another long standing bug I've seen in Sentry, where the buckets aren't defined. It's because the account could be different here, and `bucketsSelector` depends on `destinyVersionSelector` which depends on `currentAccount`.